DI-Policies.md

```diff
@@ -0,0 +1,67 @@
+         + Promotion of diversity-and-inclusion-minded vendors in my social feeds and blog
+         + Sharing love stories from underrepresented folx in my social feeds and blog
+
+   + Use my privilege, and speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website:
+      + Ensure representation of underrepresented groups in imagery across website, including blog
+      + Ensure the use of inclusive language (e.g. gender neutral terms) across website, including blog
+
+   + Monthly for social:
+      + Ensure representation of underrepresented groups in feeds, across all platforms (Instagram, Facebook, Twitter, Pinterest)
+      + Ensure the use of inclusive language extends to all social posts
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+      + Questions partners will be expected to answer include:
+         + What is your understanding and experience of Diversity and Inclusion, whether in the wedding industry or outside of it?
+         + What are the D+I (Diversity and Inclusion) policies for your own business? Do you have them written down? Can you share them with me?
+         + Do you believe love is love and support partners of all kinds, including LGBTQIA+ and interracial couples?
+         + Have you ever refused service to a couple because their partnership conflicted with your personal values?
+         + If someone were to make a racist remark while you were working with them, what would you do?
+
++ Continuing education 
+
+   + Minimum two PAID workshops a year (pay Black folx and other marginalized individuals for their work).
+
+   + Before participation in any styled shoots, workshops, or other events, inquire into their own D+I policies. 
+      + If D+I policies are nonexistent or do not align with NJP's own values on the topic, I will start a conversation to recommend the addition or amendment of a set of D+I policies. If efforts are not made to implement or amend said policies, I will not participate in, promote, or financially support the event.
+
++ Networking + referrals
+
+   + Regularly and consistently seek out, work with, and refer underrepresented vendors. This includes:
+      + Creating (year 2020) and maintaining (ongoing) a vendor referral list that is made up of 50% underrepresented folx.
+      + At least twice a year, attend networking events focused on underrepresented groups in order to connect with even more folx working in the wedding industry.
+
++ Advertising spend + outreach
+
+   + Spend at least 25% advertising dollars with sites, publications, and organizations that are focused on social justice and equity in the wedding industry.
```

Comment:
@bryanaknight I clarified this section a bit; does this make more sense now? Also, I added a percentage spend for the advertising dollars because I felt it was too open to just say "I'll spend money with these particular sites". What do you think?

cc/ @Pseudopam